SnM/SnM_Window.cpp

```diff
@@ -382,20 +382,90 @@ void ShowThemeHelper(WDL_FastString* _report, HWND _hwnd, bool _mcp, bool _sel)
 	}
 }
 
+void ShowThemeHelperREAPERv6(WDL_FastString* _report, bool _mcp, bool _sel)
+{
+	int tcpWidth = 0;
+	if (!_mcp)
+	{
+		tcpWidth = GetPrivateProfileInt("REAPER", "leftpanewid", -666, get_ini_file());
+		// leftpanewid= entry gets created first time user drags TCP/arrange divider
+		if (tcpWidth == -666)
+		{
+			MessageBox(
+				g_hwndParent, __LOCALIZE("'leftpanewid=' entry to determine TCP width doesn't exist in reaper.ini.\nPlease drag vertical TCP/arrange devider once to create it then run theme helper actions again.", "sws_mbox"), 
+				__LOCALIZE("Theme helper - Error", "sws_mbox"), MB_OK);
+			return;
+		}
+	}
+		
+	for (int trIdx = 0; trIdx <= GetNumTracks(); trIdx++)
+	{
+		{
```

Comment:
Why the extra brace/indentation level? The `for` loop is already a new scope for the variables inside.

Since both the `_mcp` and `!_mcp` branches are mostly the same, this can be shortened into (and as a bonus, this keep the existing translations of the Rv5 format string):

```cpp
static int GetTcpWidth()
{
	bool isContainer;
	HWND tcpWnd = GetTcpWnd(isContainer);
	RECT r;	GetClientRect(tcpWnd, &r);
	return r.right - r.left;
}

void ShowThemeHelperREAPERv6(WDL_FastString* _report, bool _mcp, bool _sel)
{
	const int tcpWidth = _mcp ? 0 : GetTcpWidth();

	for (int trIdx = 0; trIdx <= GetNumTracks(); ++trIdx)
	{
		MediaTrack* tr = CSurf_TrackFromID(trIdx, false);

		if (_sel && !GetMediaTrackInfo_Value(tr, "I_SELECTED"))
			continue;

		const char* trName = static_cast<const char *>(GetSetMediaTrackInfo(tr, "P_NAME", nullptr));
		if (!trName) trName = "";

		const int width = _mcp ? static_cast<int>(GetMediaTrackInfo_Value(tr, "I_MCPW")) : tcpWidth;
		const int height = static_cast<int>(GetMediaTrackInfo_Value(tr, _mcp ? "I_MCPH" : "I_TCPH"));

		if (!height) // master not visible or track hidden
			continue;

		_report->AppendFormatted(1024,
			__LOCALIZE_VERFMT("%s Track #%d '%s' : W=%d, H=%d\n", "theme_helper"),
			_mcp ? __LOCALIZE("MCP", "theme_helper") : __LOCALIZE("TCP", "theme_helper"),
			trIdx, trIdx == 0 ? __LOCALIZE("[MASTER]", "theme_helper") : trName,
			width, height
		);
	}
}
```